docker/Dockerfile

```diff
@@ -1,37 +1,21 @@
-FROM rust:slim as builder
+FROM rust:latest as builder
 
 ENV DEBIAN_FRONTEND noninteractive
 
-RUN apt-get update && \
```

Comment:
These `Dockerfile` changes are not related to the error or are they? They seem to revert my recent changes but it's not clear what's the error?

Reply:
Yes, I am studying the real reasons for CI errors, and I will remove the modifications here.